assets/css/_home.scss

```diff
@@ -10,7 +10,7 @@ html.home {
   }
 
   pre.intro {
-    font-size: 1.2em;
+    font-size: 1.2rem;
   }
 
```

Comment:
These could go either way I thought. In the context of this website, it doesn't change anything from the visual standpoint. From a code standpoint, though, I swapped for easier mental math.